carbonmark-api/src/routes/projects/get.ts

```diff
@@ -70,14 +73,24 @@ const handler = (fastify: FastifyInstance) =>
     /** Assign valid pool projects to map */
     poolProjectsData.carbonOffsets.forEach((project) => {
       if (!isValidPoolProject(project)) return;
-      const key = `${project.projectID}-${project.vintageYear}`;
+      const [standard, registryProjectId] = project.projectID.split("-");
```

Comment:
`isValidProjectId` would be great here too (similar to how you've done it in `getById.ts`